catroid/src/main/AndroidManifest.xml

```diff
@@ -81,7 +81,7 @@
         android:supportsRtl="true"
         android:requestLegacyExternalStorage="false"
         tools:ignore="UnusedAttribute"
-        tools:replace="android:label,android:allowBackup"
+        tools:replace="android:label,android:allowBackup,android:name"
```

Comment:
We should have another look at this before merging, this was for testing purposes I think and got fixed by something in paintroid?